src/frontend/apps/impress/src/features/docs/doc-editor/components/BlockNoteEditor.tsx

```diff
@@ -116,7 +126,11 @@ export const BlockNoteEditor = ({ doc, provider }: BlockNoteEditorProps) => {
         },
         showCursorLabels: showCursorLabels as 'always' | 'activity',
       },
-      dictionary: locales[lang as keyof typeof locales],
+      dictionary: {
+        ...locales[lang as keyof typeof locales],
+        multi_column:
+          multiColumnLocales[lang as keyof typeof multiColumnLocales],
```

Comment:
`multiColumnLocales` can be undefined so this code will crash the application

Reply:
Yes your right.